angular-workspace/projects/ni/nimble-angular/src/public-api.ts

```diff
@@ -97,6 +99,7 @@ export * from './testing/async-helpers';
 
 // Export items that would otherwise be exported by multiple components just from here to avoid exporting them multiple times.
 // Can be cleaned up when switching to multiple entry points, see: https://github.com/ni/nimble/issues/172
+export { BannerSeverity } from '@ni/nimble-components/dist/esm/banner/types';
```

Comment:
I believe BannerSeverity is only used by the banner directive so it should be exported just from there (not the comment above stating the exports here are for those enums that would be exported from multiple components)

Reply:
Done